docs/participant.md

```diff
@@ -168,6 +168,23 @@
         `inCall` | int | Flags whether the conflicting session is in a potential call
         `lastPing` | int | Timestamp of the last ping of the conflicting session
 
+## Resend participant emails
+
+* Method: `POST`
+* Endpoint: `/room/{token}/participants/resend-invitations`
+* Data:
+
+    field | type | Description
+    ------|------|------------
+    `participant`| string or null | v3 | User for whom to resend invitations, or null to send to all email actors
+    `attendeeId` | int or null | v3 | Attendee id can be used for guests and users
```

Comment:
so either we allow `actor id + type` or `attendee id`
but no mixing (just because older endpoints grew with legacy)

Reply:
alright, I copied it from there. I'll limit it to attendee id then.